README.md

```diff
@@ -119,8 +119,8 @@ Note that as part of the onboarding process your repository name may be normaliz
 [richer-keys-notes]: https://github.com/tc39/notes/blob/master/meetings/2019-01/jan-30.md#richer-keys-for-stage-2
 [unicode-sequence-properties]: https://github.com/tc39/proposal-regexp-unicode-sequence-properties
 [unicode-sequence-properties-notes]: https://github.com/tc39/notes/blob/master/meetings/2019-01/jan-31.md#update-on-sequence-properties-in-unicode-property-escapes
-[regex-offsets]: https://github.com/tc39/proposal-regexp-match-offsets
-[regex-offsets-notes]: https://github.com/tc39/notes/blob/master/meetings/2019-07/july-24.md#regexp-match-offsets-update
+[regex-match-indices]: https://github.com/tc39/proposal-regexp-match-indicies
```

Comment:
small typo in the URL `indicies` leads to a 404

Reply:
Good catch! Thanks!